x/gov/migrations/v3/convert.go

```diff
@@ -54,9 +54,13 @@ func ConvertToLegacyProposal(proposal v1.Proposal) (v1beta1.Proposal, error) {
 				return v1beta1.Proposal{}, err
 			}
 			legacyProposal.Content = legacyMsg.Content
+			return legacyProposal, nil
 		}
 	}
-	return legacyProposal, nil
+	// hack to fill up the content with the first message
```

Comment:
```suggestion
  // hack to fill up the content with the first message
  // this is for supporting clients that have not properly migrated to gov v1
  // https://github.com/cosmos/cosmos-sdk/issues/14334
```

Reply:
Sorry you need to lint the file and fix my typo 😬

Reply:
no problem, will do that